docs/index.md

```diff
@@ -35,11 +35,11 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
 ## Why use lakeFS? and what it can do
 
 With lakeFS data teams can stop worrying about issues that take away much of their time:
-* Roll back changes to data: recover quickly from bugs and mistakes, even on production data
-* Test and validate data before it's marked production-ready exposed to consumers
-* Travel back in time, across data versions and across collections - for reproducing and debugging data issues
+* Roll back changes to data: recover quickly from bugs and mistakes directly on your production data.
+* Test and validate your data before propagating it to data consumers.
+* Travel back in time, across data versions and across data collections - for reproducing and debugging data issues.
 
-Since lakeFS is compatible with the S3 API, all popular applications will work without modification, by simply adding the branch name to the object path:
+Since lakeFS is compatible with the S3, Azure Blog and GCP object storage API, all popular applications will work without modification, by simply adding the branch name to the object path:
```

Comment:
Unfortunately this is not true - while we do support all cloud platforms as the underlying storage - our API is compatible with S3 but not with the other two.

Reply:
thanks, updating!